apps/asap-server/test/controllers/research-outputs.test.ts

```diff
@@ -609,6 +614,50 @@ describe('ResearchOutputs controller', () => {
       });
     });
   });
+
+  describe('create', () => {
+    afterEach(() => {
+      expect(nock.isDone()).toBe(true);
+    });
+
+    afterEach(() => {
+      nock.cleanAll();
+    });
+    test('creating the research output should return the id from squidex rest', async () => {
+      const researchOutput = { type: 'test' } as unknown as ResearchOutput;
+      nock(config.baseUrl)
+        .post(`/api/content/${config.appName}/research-outputs?publish=false`, {
+          type: { iv: 'test' },
+          asapFunded: { iv: 'Not Sure' },
+          usedInAPublication: { iv: 'Not Sure' },
+        })
+        .reply(201, { id: 90210 });
+
+      const id = await researchOutputs.create(researchOutput);
+      expect(id).toEqual(90210);
+    });
+    test('should throw when fails to create the research output - 400', async () => {
+      const researchOutput = { type: 1 } as unknown as ResearchOutput;
+      nock(config.baseUrl)
+        .post(`/api/content/${config.appName}/research-outputs?publish=false`)
+        .reply(400);
+
+      await expect(researchOutputs.create(researchOutput)).rejects.toThrow(
+        'Bad Request',
```

Comment:
maybe we should be more expllicit with errors in our controllers? for instance maybe we should capture the HTTP errors and rethrow with a more meaningful message like 'Research Output failed to create', you could attach the original error to the stack im sure

Reply:
We were trying to follow the pattern from the other controllers (to be more consistent). We didn't want to introduce a new more explicit error handling pattern.

Reply:
Right, i actually think some of them throw more specific errors, or maybe its the parsers or something. Sure thats fine